polaris-react/src/components/Inline/Inline.tsx

```diff
@@ -21,10 +26,13 @@ export interface InlineProps {
    * @default 'center'
    */
   blockAlign?: BlockAlign;
-  /** The spacing between elements
+  /** The spacing between elements. Accepts a spacing token or an object of spacing tokens for different screen sizes.
    * @default '4'
+   * @example
+   * gap='2'
+   * gap={{xs: '2', md: '4'}}
```

Comment:
Can we give examples at all breakpoints?
```suggestion
   * gap={{xs: 1, sm: 2, md: 3, lg: 4, xl: 5}}
```

Really like using `@example` here 👍